src/style.css

```
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-size: 16px;
+  padding-top: 20px;
+  margin-top: 0;
+  background-color: #fff;
+}
+
+.search_bar {
+  display: block;
+}
+
+.search_bar .search_field {
```

```suggestion
.search_field {
```
Не увеличивай специфичность селекторов